protos/api.v25.proto

```diff
@@ -158,3 +160,23 @@ message Namespace {
   string name = 1;
   uint64 id = 2;
 }
+
+message InitiateSnapshotStreamRequest {}
+
+message InitiateSnapshotStreamResponse {
+    repeated uint32 groups = 1;
```

Comment:
Should we keep this as a struct in case we decide to add more fields or information regarding each group in future?